package.json

```diff
@@ -37,6 +37,7 @@
     "react": "18.2.0",
     "react-dom": "18.2.0",
     "react-timer-hook": "^3.0.6",
+    "superjson": "^1.12.3",
```

Comment:
it looks like superjson is mostly useful for serializing/deserializing Dates? would be good to include a comment somewhere documenting why it's used

Reply:
Added a comment to backend/trpc.ts.